### DISCUSSION TITLE
Help: replacing featured-background.jpg has no effect.

### DISCUSSION THREAD

**Discussion by iani** [google/docsy#1322] [General]
Hello, 

I am trying to replace the default featured background image.
I put my own image in content/en/featured-background.jpg
I restarted the hugo server.  The page background  remains the same, 
and the featured-background images in the resources/_gen/images directory 
also remain the same.

What am I doing wrong?

Thank you,

Iannis Zannos

**Comment by Sidon** [comment]
I did the following test: I removed all files named featured-background.jpg, and the background continues appearing when the site is loaded. I wonder if this isn´t loading from the module (theme) in a way like hardcode. :thinking:

**Comment by Sidon** [comment]
@iani , I found out what my problem was. Maybe it can help you.
In my case, in fact, I didn´t delete the original files but just renamed t to `featured-old ` (or something like that). After an investigation of the source code, I inferred that the file is loaded through searching by the name `**featured*`,  then I renamed it to `old-featured..`. and it's worked.

**Comment by iani** [comment]
Hello, 

thank you for answering. In fact my problem was 2-fold
1. that I did not  replace the featured pic file from the about page as well. 
2. that I did the same mistake as you with the filename.

Thanks for answering!

Iannis Zannos
